### PR TITLE
SRE-2183 - Added updated mongo db dashboard.

### PR DESCRIPTION
**Pull request by thangarajbluescape**
Please review and merge this PR:
Changes already applied in sre1 cluster.
https://grafana.b-sre1.dev.bluescape.io/d/AyWQt9jWk/mongodb?orgId=1&refresh=5s
image

### CODE REVIEW

**Review by dperumal-bs — APPROVED** [review]
LGTM

File: charts/bluescape-monitoring-dashboards/templates/96_mongodb/dashboard.yaml [hunk 1]
```diff
@@ -7,7 +7,8 @@ metadata:
   labels:
      {{- include "bluescape-monitoring-dashboards.labels" . | nindent 4 }}
 spec:
-  json: ""
+  json: 
   configMapRef:
     name: mongodb-dashboard
     key: mongodb-dashboard.json
+  name: mongodb-dashboard
```
Comment (ashwinrajadurai3) [inline charts/bluescape-monitoring-dashboards/templates/96_mongodb/dashboard.yaml:14]: Hi @thangarajbluescape Do we need this name field inside spec:? We already have it in metadata & also in configmapref.

**Review by ashwinrajadurai3 — CHANGES_REQUESTED** [review]

File: charts/bluescape-monitoring-dashboards/templates/96_mongodb/dashboard.yaml (hunk 1 above)
Comment (ashwinrajadurai3) [reply charts/bluescape-monitoring-dashboards/templates/96_mongodb/dashboard.yaml:14]: LGTM then!

**Review by ashwinrajadurai3 — APPROVED** [review]
LGTM

**Pull request merged**